docs/getting_started.rst

```diff
@@ -128,19 +134,23 @@ like to run ``cfme_tests`` on his/her system, here are some tools and tips that
 started as quickly as possible:
 
 * ``cfme_tests`` expects an appliance, with an IP visible to machine with ``cfme_tests`` running
-  
+
   * If this is not the case (eg. CFME behind NAT, a container, whatever), you have to specify the
     ``base_url`` in configuration with a port, which is quite obvious, but people tend to forget
-    ``cfme_tests`` also uses SSH and Postgres extensively, therefore you need to have those services
+    ``cfme_tests`` also uses SSH and Postgres extensively, therefore you MUST have those services
     accessible and ideally on the expected ports. If you don't have them running on the expected
-    ports, you have to specify them manually using ``--port-ssh`` and ``--port-db`` command-line
-    parameters.
-
-* ``cfme_tests`` also expects that the appliance it is running against is configured. By
-  'configured', we mean the database is set up and seeded (therefore UI running), database
-  permissions loosened so ``cfme_tests`` can access it and a couple of other fixes. Check out
-  :py:meth:`utils.appliance.IPAppliance.configure`, and subsequent method calls.
-  
+    ports, you MUST specify them manually using ``--port-ssh`` and ``--port-db`` command-line
```

Comment:
@mfalesni just a note that we should probably incorporate this into the yaml and ensure it's happening at the "Appliance" object level.